src/aspire/basis/fspca.py

```diff
@@ -536,7 +536,7 @@ def rotate(self, coef, radians, refl=None):
     def calculate_bispectrum(
         self, coef, flatten=False, filter_nonzero_freqs=False, freq_cutoff=None
     ):
-        if coef.dtype == real_type(self.dtype):
+        if coef.dtype not in (np.complex64, np.complex128):
```

Comment:
Are we no longer requiring that dtype precision be maintained?

Reply:
Actually I did not mean to commit this! I was printing something here while I was checking some float change that wasn't needed and only reverted half the debugging change.  I was rushing, sorry.  

Reverted.

Thank you.